### PR TITLE
Target Specta pod at Xcode8 branch

### PR DESCRIPTION
**Pull request by mschouest**
Specta master branch does not work with xcode8, but demo app requires it. I've pointed the podfile to the xcode8 compatible branch on github.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -8,7 +8,6 @@ target "Hakawai" do
 end
 
 target "HakawaiTests" do
-  pod 'Specta' 
-  pod 'Expecta'
+  pod 'Specta', :git => 'https://github.com/specta/specta.git', :branch => 'xcode_8'
+  pod 'Expecta', :git => 'https://github.com/specta/expecta.git'
```
Comment (dgattey) [inline Podfile]: Is this change necessary?

Comment (mschouest) [reply Podfile]: the master branch of Specta does not compile on xcode8. Additionally, when running pod install without pointing to the git for both specta and expecta, cocoapods can't find the libraries for some reason. Pointing directly fixes this.

Comment (jmkk) [reply Podfile]: We should plan on deprecating these since they are removed from Cocoapods and no longer maintained.
